cmd/serve.go

```diff
@@ -62,6 +64,22 @@ func run() error {
 	}
 	log.Debug().Str("status", cfg.Jira.Issue.Status).Msg("Configured default status found ✓")
 
-	s := server.New(&cfg, jiraClient)
+	patcher, err := newTestedPatcher()
+	if err != nil {
+		return err
+	}
+
+	s := server.New(&cfg, jiraClient, patcher)
 	return s.Serve()
 }
+
+func newTestedPatcher() (patch.Patcher, error) {
+	patcher, err := patch.New(&cfg)
+	if err != nil {
+		return patch.Patcher{}, err
+	}
+	if err := patcher.TestGitHubConnection(context.TODO()); err != nil {
```

Comment:
Just to make sure its intentional: wanna leave TODO context or pass one?

Reply:
I opted for using `context.TODO()` as I don't want to refactor the codebase for handling contexts yet. I want to leave that to a future PR to not increase the scope of this one